### DISCUSSION TITLE
Searchable menus

### DISCUSSION THREAD

**Discussion by hawkrives** [StoDevX/AAO-React-Native#5556] [Ideas]
From a user:

> It would be nice to be able to search within a menu (e.g. The Cage) to find items by name!

(This might fit under an existing issue; "Related Issues" didn't find anything.)

Thoughts?

**Comment by drewvolz** [comment]
The closest thing I can find is https://github.com/StoDevX/AAO-React-Native/issues/509

> Add searching/filtering to some ListViews
> Immediate thoughts: menus, dictionary, orgs, hours(?)

**Comment by drewvolz** [comment]
I am a big fan of searching on the menu. Quicker than parsing station > items with the eyes.
